backend/service/aws/dynamodb.go

```diff
@@ -57,3 +89,63 @@ func newProtoForGlobalSecondaryIndex(index types.GlobalSecondaryIndexDescription
 		ProvisionedThroughput: currentCapacity,
 	}
 }
+
+func isValidIncrease(client *regionalClient, current *types.ProvisionedThroughputDescription, target types.ProvisionedThroughput) error {
+	// check for targets that are lower than current (can't scale down)
+	if *current.ReadCapacityUnits > *target.ReadCapacityUnits {
+		return status.Errorf(codes.FailedPrecondition, fmt.Sprintf("Target read capacity [%d] is lower than current capacity [%d]", *target.ReadCapacityUnits, *current.ReadCapacityUnits))
```

Comment:
nit: since you're using errorf you don't need sprintf, e.g.

```suggestion
		return status.Errorf(codes.FailedPrecondition, "Target read capacity [%d] is lower than current capacity [%d]", *target.ReadCapacityUnits, *current.ReadCapacityUnits)
```